src/Shared/Recruit.Vacancies.Client/Application/Validation/Fluent/FluentVacancyValidator.cs

```diff
@@ -135,7 +135,37 @@ private void ValidateOrganisation()
                     .RunCondition(VacancyRuleSet.TradingName)
                     .WithRuleId(VacancyRuleSet.TradingName)
             );
-            
+
+            When(v => v.SourceOrigin == SourceOrigin.EmployerWeb && v.EmployerNameOption == EmployerNameOption.Anonymous, () =>
+                RuleFor(x => x.EmployerName)
+                    .NotEmpty()
+                    .WithMessage("You must provide a description")
+                    .WithErrorCode("401")
+                    .MaximumLength(100)
+                    .WithMessage("The description must not exceed {MaxLength} characters")
+                    .WithErrorCode("402")
```

Comment:
Are these error codes supposed to be the same as the ones below?

Also isn't the `description` in the error message quite generic in relation to the on screen field.

Reply:
👍 for the error codes.
As for `description`. Maybe, but its the content required by the story though.